CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -591,18 +562,25 @@ public function emailReceipt(&$params) {
     // assign payment info here
     $paymentConfig['confirm_email_text'] = CRM_Utils_Array::value('confirm_email_text', $params);
     $this->assign('paymentConfig', $paymentConfig);
+
+    $this->assign('totalAmount', $this->_amtTotal);
```

Comment:
just a code shift as there always be ```$this->_amtTotal```, so I just moved it from the IF-ELSE code below